source/spelling_wordlist.txt

```diff
@@ -6,6 +6,7 @@ accessor
 accessors
 backend
 backends
+const
```

Comment:
Khronos Guys asked us to keep const as in code formatting.

Reply:
Removed with https://github.com/imvucic/SYCL_Reference/pull/20/commits/7b6e03dc9cca91697627588f7f7cab406545983a